$OpenBSD: patch-server_c,v 1.2 2020/03/27 01:32:06 kn Exp $

Use libevent2 not libevent from base.
Add missing inet promise.

Index: server.c
--- server.c.orig
+++ server.c
@@ -24,7 +24,7 @@
 #include <sys/wait.h>
 
 #include <errno.h>
-#include <event.h>
+#include <event2/event.h>
 #include <fcntl.h>
 #include <signal.h>
 #include <stdio.h>
@@ -169,7 +169,7 @@ server_start(struct event_base *base, int lockfd, char
 
 #ifdef __OpenBSD__
 	if (pledge("stdio rpath wpath cpath fattr unix getpw recvfd proc exec "
-	    "tty ps", NULL) != 0)
+	    "tty ps inet", NULL) != 0)
 		fatal("pledge failed");
 #endif
 
